enterprise/backend/src/metabase_enterprise/serialization/v2/serialize.clj

```diff
@@ -0,0 +1,14 @@
+(ns metabase-enterprise.serialization.v2.serialize
+  "Top-level functionality for serializing a Metabase appdb into YAML files for import into another.
```

Comment:
I would suggest separating out the task of `stuff in app DB -> YAML` into two separate tasks

1. `stuff in app DB -> serializable form(s)` (e.g. generic Clojure data structures)
2. serializable form(s) -> YAML

(and vice versa for deserialization).

That would make things a lot easier to test I think and keep concerns separate. Also not saying we will want to support any formats other than YAML but serialization is not inherently a YAML thing so it seems like tying it in to YAML is unnecessary coupling

Reply:
That's actually how it's designed; there's no YAML anywhere except the filename suffixes. That could be dropped from the filenames so that any format would work.

Reply:
Done.